test/test_reporter.rb

```diff
@@ -26,6 +26,7 @@ def default_block
 
   # Shared method that creates a Results with 1 retained object using options provided
   def create_report(options = {}, &profiled_block)
+    GC.start
```

Comment:
note you may need to do this 3 times or even more to fully collect stuff.

Reply:
Yep. I am still unsure where even to put it. I am very confused by the fact that only `test_counts` seems to fail.

Reply:
Ahh, it’s the only test case running `create_report` with its `default_block` *and* making assertions about total_retained.

I am leaning towards just skipping that assertion for Ruby 2.6 😬